<ui version="4.0" >
 <class>DownloadMapDialog</class>
 <widget class="QDialog" name="DownloadMapDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Download</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Location</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="FromBookmark" >
          <property name="text" >
           <string>Bookmark</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Bookmarks" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="FromView" >
        <property name="text" >
         <string>Current view</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="FromviewAndAdd" >
          <property name="text" >
           <string>Current view and add as bookmark</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NewBookmark" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="FromMap" >
        <property name="text" >
         <string>From the map below (map provided by the OpenStreetMap project)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="IncludeTracks" >
     <property name="text" >
      <string>Also download raw GPS tracks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DownloadMapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>228</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel" >
     <x>241</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DownloadMapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>284</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel" >
     <x>283</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FromBookmark</sender>
   <signal>toggled(bool)</signal>
   <receiver>Bookmarks</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>76</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel" >
     <x>220</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FromviewAndAdd</sender>
   <signal>toggled(bool)</signal>
   <receiver>NewBookmark</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>116</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel" >
     <x>431</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
